2023-03-31T17:53:40.997Z	INFO	controllers.CassandraDatacenter.cassandradatacenter_controller.controller.cassandradatacenter-controller	handler::CreateReconciliationContext	{"reconciler group": "cassandra.datastax.com", "reconciler kind": "CassandraDatacenter", "name": "test-cluster", "namespace": "cass-operator", "namespace": "cass-operator"}
2023-03-31T17:53:40.997Z	INFO	controllers.CassandraDatacenter.cassandradatacenter_controller.controller.cassandradatacenter-controller	handler::calculateReconciliationActions	{"reconciler group": "cassandra.datastax.com", "reconciler kind": "CassandraDatacenter", "name": "test-cluster", "namespace": "cass-operator", "namespace": "cass-operator", "datacenterName": "test-cluster", "clusterName": "cluster1"}
2023-03-31T17:53:40.997Z	INFO	controllers.CassandraDatacenter.cassandradatacenter_controller.controller.cassandradatacenter-controller	reconcile_services::ReconcileHeadlessServices	{"reconciler group": "cassandra.datastax.com", "reconciler kind": "CassandraDatacenter", "name": "test-cluster", "namespace": "cass-operator", "namespace": "cass-operator", "datacenterName": "test-cluster", "clusterName": "cluster1"}
2023-03-31T17:53:40.998Z	INFO	controllers.CassandraDatacenter.cassandradatacenter_controller.controller.cassandradatacenter-controller	reconcile_endpoints::CheckAdditionalSeedEndpoints	{"reconciler group": "cassandra.datastax.com", "reconciler kind": "CassandraDatacenter", "name": "test-cluster", "namespace": "cass-operator", "namespace": "cass-operator", "datacenterName": "test-cluster", "clusterName": "cluster1"}
2023-03-31T17:53:40.998Z	ERROR	controllers.CassandraDatacenter.cassandradatacenter_controller.controller.cassandradatacenter-controller	Could not set additional seeds for endpoints for additional seed service	{"reconciler group": "cassandra.datastax.com", "reconciler kind": "CassandraDatacenter", "name": "test-cluster", "namespace": "cass-operator", "namespace": "cass-operator", "datacenterName": "test-cluster", "clusterName": "cluster1", "error": "lookup : no such host"}
github.com/k8ssandra/cass-operator/controllers/cassandra.(*CassandraDatacenterReconciler).Reconcile
	/workspace/controllers/cassandra/cassandradatacenter_controller.go:143
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.0/pkg/internal/controller/controller.go:114
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.0/pkg/internal/controller/controller.go:311
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.0/pkg/internal/controller/controller.go:266
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.0/pkg/internal/controller/controller.go:227
2023-03-31T17:53:40.998Z	ERROR	controllers.CassandraDatacenter	calculateReconciliationActions returned an error	{"cassandradatacenter": "cass-operator/test-cluster", "requestNamespace": "cass-operator", "requestName": "test-cluster", "loopID": "7fff3e9d-74cc-4558-bde5-15a9d2742cbb", "error": "lookup : no such host"}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.0/pkg/internal/controller/controller.go:311
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.0/pkg/internal/controller/controller.go:266
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.0/pkg/internal/controller/controller.go:227
2023-03-31T17:53:40.998Z	INFO	controllers.CassandraDatacenter.cassandradatacenter_controller.controller.cassandradatacenter-controller	lookup : no such host	{"reconciler group": "cassandra.datastax.com", "reconciler kind": "CassandraDatacenter", "name": "test-cluster", "namespace": "cass-operator", "reason": "ReconcileFailed", "eventType": "Warning"}
2023-03-31T17:53:40.998Z	INFO	controllers.CassandraDatacenter	Reconcile loop completed	{"cassandradatacenter": "cass-operator/test-cluster", "requestNamespace": "cass-operator", "requestName": "test-cluster", "loopID": "7fff3e9d-74cc-4558-bde5-15a9d2742cbb", "duration": 0.001454533}
2023-03-31T17:53:40.998Z	ERROR	controllers.CassandraDatacenter.cassandradatacenter_controller.controller.cassandradatacenter-controller	Reconciler error	{"reconciler group": "cassandra.datastax.com", "reconciler kind": "CassandraDatacenter", "name": "test-cluster", "namespace": "cass-operator", "error": "lookup : no such host"}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.0/pkg/internal/controller/controller.go:227
2023-03-31T17:53:40.998Z	DEBUG	events	Warning	{"object": {"kind":"CassandraDatacenter","namespace":"cass-operator","name":"test-cluster","uid":"55425bf5-18b2-4dab-8642-bc655f77e923","apiVersion":"cassandra.datastax.com/v1beta1","resourceVersion":"2220"}, "reason": "ReconcileFailed", "message": "lookup : no such host"}

